src/view/DialogNew.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {FormEvent} from "react";
+import {Innholdstittel, Normaltekst} from "nav-frontend-typografi";
+import {Input, Textarea} from "nav-frontend-skjema";
+import {Hovedknapp} from "nav-frontend-knapper";
+import useFieldState from "../utils/useFieldState";
+
+import './Dialog.less';
+
+function validerMelding(melding: string): string | null {
+    if (melding.length === 0) {
+        return "Melding må ha innhold.";
+    } else if (melding.length < 10) {
+        return "Meldingen må være lengre enn 10 tegn.";
+    } else {
+        return null;
+    }
+}
+function validerTema(tema: string): string | null {
```

Comment:
Hva om tema kun er et space? Det vil være rart, og burde kanskje ikke være lov. Foreslår en validering for det også

Reply:
Har lagt til følgende sjekk:
else if (!melding.replace(/\s/g, '').length) {
        return "Meldingen må ha lesbart innhold.";
    }